primitives/api/test/tests/decl_and_impl.rs

```diff
@@ -46,6 +46,8 @@ decl_runtime_apis! {
 		fn same_name();
 		#[changed_in(2)]
 		fn same_name() -> String;
+		#[api_version(3)]
```

Comment:
I abducted this test to check if the happy case works. Proper testing should be implemented.